nuget/libcpr.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>libcpr</id>
+    <version>$VERSION$</version>
+    <title>C++ Requests: Curl for People</title>
+    <authors>Simon Berger</authors>
+    <owners></owners>
```

Comment:
Please add "Fabian Sauter", "Kilian Traub" and "many other libcpr contributors" as owners